_data/blogs.yml

```diff
@@ -137,3 +137,10 @@
   title: Interview with a FOSSASIA Contributer | Ploypiti Piyaprapan
   img: aditya-blog-2.jpg
   author_name: Aditya Dutt
+  
+- url: https://wordpress.com/view/patriciascodeintime.wordpress.com 
+  title: My journey with Google Code-in 
+  img: Part 1 of blog.jpg, Part 2 of blog.jpg, Part 3 of blog.jpg, Part 4 of blog.jpg, Part 5 of blog.jpg
```

Comment:
@lillietricianijiiro This should contain a single image name, that has been uploaded to the ```img/blog/``` folder.

Reply:
@lillietricianijiiro see this link https://lillietricianijiiro.github.io/gci17.fossasia.org/
Your blog image is not being displayed because you have not written the name of the image correctly and you also need to add the image in the  img/blog/ folder